zeebe/docker/utils/startup.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/sh -xeu
 
 HOST=$(hostname -i)
+ZEEBE_SET_SEED_ENV_VARS=${ZEEBE_SET_SEED_ENV_VARS:-true}
 
 if [ "$ZEEBE_STANDALONE_GATEWAY" = "true" ]; then
-    export ZEEBE_GATEWAY_NETWORK_HOST=${ZEEBE_GATEWAY_NETWORK_HOST:-${HOST}}
+    if [ "$ZEEBE_SET_SEED_ENV_VARS" = "true" ]; then
+        export ZEEBE_GATEWAY_NETWORK_HOST=${ZEEBE_GATEWAY_NETWORK_HOST:-${HOST}}
+    fi
     export ZEEBE_GATEWAY_CLUSTER_HOST=${ZEEBE_GATEWAY_CLUSTER_HOST:-${ZEEBE_GATEWAY_NETWORK_HOST}}
 
     exec /usr/local/zeebe/bin/gateway
```

Comment:
The same should be done for `ZEEBE_BROKER_*` vars.

Reply:
Only `ZEEBE_GATEWAY_NETWORK_HOST` needs to change to `0.0.0.0`. I don't understand why we need to add other env vars into the if statement as well.